.github/workflows/dependency-upgrade.yaml

```diff
@@ -13,17 +13,65 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        dependency: ["Stargate", "Medusa", "Reaper"]
+        dependency: ["Stargate", "Medusa", "Reaper", "cass-operator", "k8ssandra-operator", "medusa-operator", "reaper-operator"]
         include:
           - dependency: Stargate
             version_path: ".stargate.version"
-            docker_hub_url: "https://registry.hub.docker.com/v2/repositories/stargateio/stargate-3_11/tags?page_size=25"
+            values_file: "charts/k8ssandra/values.yaml"
+            chart_version_path: ""
```

Comment:
This parameter is either empty for non-operator dependencies, or `.version` for the operator ones. I think you could get rid of it completely, since all usages of this parameter are done for operator dependencies only, and the whole thing is protected by an if block ` if [ "${{ matrix.operator }}" == "true" ]`.

Reply:
👍